app/services/ccms/attribute_value_generator.rb

```diff
@@ -92,6 +87,42 @@ def app_amendment_type(_options)
       @legal_aid_application.used_delegated_functions? ? 'SUBDP' : 'SUB'
     end
 
+    def client_owns_vehicle(_options)
+      @legal_aid_application.own_vehicle? ? 'true' : 'false'
+    end
+
+    def client_owed_money(_options)
+      @legal_aid_application.other_assets_declaration.money_owed_value.nil? ? 'false' : 'true'
```

Comment:
There is a lot of repetition here of the pattern `x.nil? ? 'true' : 'false'`. Firstly, if we can return Booleans, could this simply be:
```ruby
@legal_aid_application.other_assets_declaration.money_owed_value.present?
```
If the pattern used is really the only way to do it, I'd suggest wrapping in a method:
```ruby
def not_nil?(value)
  value.nil? ? 'false' : 'true'
end

def client_owed_money(_options)
  not_nil? @legal_aid_application.other_assets_declaration.money_owed_value
end
```
etc.

Reply:
I've just had a bit of a play:
```ruby
x = Nokogiri::XML('<body><foo></foo><bar></bar></body>')

f = x.at_xpath '/body/foo'
f.content = 'true'
f.set_attribute 'bol', 'true'

b = x.at_xpath '/body/bar'
b.content = true
b.set_attribute 'bol', true

puts x.to_s
```
Outputs:
```xml
<?xml version="1.0"?>
<body>
  <foo bol="true">true</foo>
  <bar bol="true">true</bar>
</body>
```
As you can see setting `true` or `"true"` results in the same XML output.

Reply:
So remove all the text "true" and "false" and just use boolean operations `.nil?`, `.blank?` and `.present?` etc.

Reply:
yes all works, thank you much cleaner